templates/default/components/Nav.js

```diff
@@ -14,6 +14,36 @@ function GitHubIcon(props) {
   );
 }
 
+function NavbarTitle() {
+  // include option to define different text from title (if needed) in config
+  const title = siteConfig.navbarTitle?.text || siteConfig.title
```

Comment:
Personally, i would be strict and say if you use navbarTitle you have to set title in their too i.e. this is:

```
if (navbarTitle) {
	use stuff from navbarTitle
} else {
      use siteConfig.title
}
```

Why? Explicitness is better than magic!